src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/FullBlockSectionProcessor.scala

```diff
@@ -0,0 +1,88 @@
+package org.ergoplatform.nodeView.history.storage.modifierprocessors
+
+import io.iohk.iodb.ByteArrayWrapper
+import org.ergoplatform.modifiers.history.{ADProofs, BlockTransactions, Header}
+import org.ergoplatform.modifiers.{BlockSection, ErgoFullBlock, ErgoPersistentModifier}
+import scorex.core.consensus.History.ProgressInfo
+import scorex.core.consensus.ModifierSemanticValidity.Invalid
+import scorex.core.utils.ScorexEncoding
+import scorex.core.validation.{ModifierValidator, RecoverableModifierError, ValidationResult}
+
+import scala.util.{Failure, Try}
+
+/**
+  * Trait, that implements BlockSectionProcessor interfaces for regimes, where node download
+  * and process full blocks
```

Comment:
Trait that implements BlockSectionProcessor interfaces for regimes where the node is downloading and processing full blocks.